### PR TITLE
feat(curriculum): Add English tasks B1 Block 2

### PR DESCRIPTION
**Pull request by anastasiiauk**
Checklist:

- [x] I have read and followed the [contribution guidelines](https://contribute.freecodecamp.org).
- [x] I have read and followed the [how to open a pull request guide](https://contribute.freecodecamp.org/how-to-open-a-pull-request/).
- [x] My pull request targets the `main` branch of freeCodeCamp.
- [x] I have tested these changes either locally on my machine, or GitPod.

### CODE REVIEW

**Review by nieldakarla — CHANGES_REQUESTED** [review]
Hey @anastasiiauk ! Thank you for creating the block 2. I've started reviewing the block but I noticed there are many tasks that are not following the structures and order we usually follow on the curriculum. Also, some tasks are using audio references that don't exist in the dialogue. 
I've talked to @DanielRosa74 and, since he was already going to work on this PR to add the animations, we agreed he'll go over the tasks and do the fixing we need. Once it's done we'll get back to you and show what was changed so you can apply to future PRs. Is that ok with you?

File: curriculum/challenges/english/24-b1-english-for-developers/learn-how-to-talk-about-past-experiences/6705605c7fb8f619e0068634.md
[the remark this answers is not in the document]
Comment (anastasiiauk) [reply curriculum/challenges/english/24-b1-english-for-developers/learn-how-to-talk-about-past-experiences/6705605c7fb8f619e0068634.md:1]: Tasks 22-24 are based on the Wh- questions. In my opinion, they should come together as they cover one grammar section. Using them in the beginning of the block wouldn’t make much sense as well, as I would cover whole dialogue.

File: curriculum/challenges/english/24-b1-english-for-developers/learn-how-to-talk-about-past-experiences/67053619dd65bd1459449bf0.md [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+
+James saw something odd. Use the past tense form since the action has already occurred.
+
+# --explanation--
+
+To `notice` means to become aware of something or to observe it. In the context of debugging, developers notice unusual patterns or errors in code or logs. For example:
+
+- `I didn’t notice the bug in the code.` - You failed to see the error.
+
+- `They will notice the changes in the interface.` - They will become aware of the modifications.
```
[the remark this answers is not in the document]
Comment (anastasiiauk) [reply curriculum/challenges/english/24-b1-english-for-developers/learn-how-to-talk-about-past-experiences/67053619dd65bd1459449bf0.md:37]: What about irregular verbs? Wouldn't it be better to use different forms to show the difference? For example, `apply` and `applied`

[the remark this answers is not in the document]
Comment (anastasiiauk) [reply curriculum/challenges/english/24-b1-english-for-developers/learn-how-to-talk-about-past-experiences/67053619dd65bd1459449bf0.md:37]: Or irregular plural nouns, like `index` and `indices`

**Pull request closed without merging**